### ISSUE TITLE
$export does not work

### ISSUE THREAD

**Issue by kiwisky** [hapifhir/hapi-fhir-jpaserver-starter#364]
I am deploying hapi-fhir-jpaserver-starter to an application server with Apache Tomcat.
If following setting are set:
bulk_export_enabled: true

the jpaserver can not be started. 
I disable bulk_export_enabled, the jpaserver is OK to start but I got following error from [base]/$export or Patient/$export.
ponse Body
{
"resourceType": "OperationOutcome",
"text": {
"status": "generated",
"div": "<div xmlns="[http://www.w3.org/1999/xhtml\">](http://www.w3.org/1999/xhtml%5C%22%3E)

Operation Outcome
<table border="0"><td style="font-weight: bold;">ERROR[]
Invalid request: The FHIR endpoint on this server does not know how to handle GET operation[$export] with parameters [[]]
\n\t\t\t\n\t\t\n\t"
},
"issue": [ {
"severity": "error",
"code": "not-supported",
"diagnostics": "Invalid request: The FHIR endpoint on this server does not know how to handle GET operation[$export] with parameters [[]]"
} ]
}

**Comment by kiwisky** [comment]
The reason is that the configuration "bulk_export_enabled" in application.yaml must be aligned.

**Comment by brianraila** [comment]
@kiwisky this doesn't seem to sort the issue.
I have an application.yaml im using but still getting the issue.
[https://github.com/brianraila/hapifhir-docker/blob/main/application.yaml ](https://github.com/brianraila/hapifhir-docker/blob/main/application.yaml ) on line 112.
Kindly assist

**Comment by billkuojhgf** [comment]
@brianraila The $export request is working successfully on my hapi.
FYI, this is my configs.
[https://github.com/billkuojhgf/Model-on-fhir/blob/develop/Configs/another.application.yaml](https://github.com/billkuojhgf/Model-on-fhir/blob/develop/Configs/another.application.yaml)

**Comment by brianraila** [comment]
Thanks 👍🏿

**Comment by brianraila** [comment]
@billkuojhgf Still unable to do that.
Getting this 
```
{
  "resourceType": "OperationOutcome",
  "text": {
    "status": "generated",
    "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\"><h1>Operation Outcome</h1><table border=\"0\"><tr><td style=\"font-weight: bold;\">ERROR</td><td>[]</td><td><pre>Invalid request: The FHIR endpoint on this server does not know how to handle GET operation[$export] with parameters [[]]</pre></td>\n\t\t\t</tr>\n\t\t</table>\n\t</div>"
  },
  "issue": [ {
    "severity": "error",
    "code": "not-supported",
    "diagnostics": "Invalid request: The FHIR endpoint on this server does not know how to handle GET operation[$export] with parameters [[]]"
  } ]
}
```

**Comment by billkuojhgf** [comment]
Hmmm... interesting.
I deploy hapi server with Docker and Jetty with this application.yaml, and both of them works fine on it.
Have you tried out deploying your project with Docker or Jetty? If these two deploy methods all works fine with this configuration, then maybe the problem is because of the Tomcat.
BTW, I just looked at your hapi server, and I found out that your HAPI is running at version 5.7.0, can that be the reason why the server doesn't know how to handle this operation? I'm using hapi server with version 6.0.1, and I don't remember whether the previous version supports FHIR Bulk export operaions yet.
